src/TwigExtension.php

```diff
@@ -22,4 +24,94 @@ public function is_instanceof($value, string $type): bool
       || (\function_exists($func = 'is_'.$type) && $func($value))
       || $value instanceof $type;
   }
-}
\ No newline at end of file
+
+  /**
+   * Returns entity ids of entities with matching title/name/label.
+   *
+   * Supported entity types are node, taxonomy_term, group, and user.
+   *
+   * @param  string  $label
+   *   The entity label that we're looking for
+   * @param  string  $entity_type
+   *   The entity type.
+   * @param  string  $bundle_identifier
+   *   The entity bundle (may be empty)
+   * @param  bool  $check_access
+   *   Whether to check for entity access permission (default TRUE).
+   * @param  int  $limit
+   *   Restrict to number of results. Capped at no more than 100.
+   * @param  null  $langcode
+   *   (optional) For which language the entity should be rendered, defaults to
+   *   the current content language.
+   *
+   * @return null|array
+   *   An array of render arrays for the entities found, or NULL if the entity does not exist.
+   */
+  public function sbf_entity_ids_by_label(string $label, string $entity_type, string $bundle_identifier = '', $check_access = TRUE, $limit = 1, $langcode = NULL): ?array {
```

Comment:
This is much changed now. I removed the access check logic since apparently entityQuery include access check by default, and I'm convinced that there is no need to override that.

Reply:
Great! Resolved